tests/test_aiohttp_error_handlers.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+from aiohttp import web
+
+from apistrap.errors import ApiClientError, ApiServerError
+
+
+@pytest.fixture()
+def app_with_errors(aiohttp_apistrap):
+    app = web.Application(debug=True)
+    routes = web.RouteTableDef()
+
+    @routes.get('/client_error')
+    def view_1(request):
+        raise ApiClientError()
+
+    @routes.get('/server_error')
+    def view_2(request):
+        raise ApiServerError()
+
+    @routes.get('/internal_error')
+    def view_3(request):
+        raise RuntimeError('Runtime error occurred')
+
+    app.add_routes(routes)
+    aiohttp_apistrap.init_app(app)
+    yield app
+
+
+@pytest.fixture()
+def app_with_errors_in_production(aiohttp_apistrap):
```

Comment:
Seems like a clone of `app_with_errors` - could we use some kind of [parametrization](https://docs.pytest.org/en/latest/fixture.html#fixture-parametrize)?
The following tests are very similar except the presence `debug_data`

Reply:
I'll probably put the routes in a separate fixture, that should eliminate most of the duplication.